### PR TITLE
Branch UI change

### PR DESCRIPTION
**Pull request by hjuntan**

### CODE REVIEW

File: src/main/java/seedu/address/ui/PersonCard.java [hunk 1]
```diff
@@ -57,12 +65,45 @@ public PersonCard(Person person, int displayedIndex) {
...
+    /**
+     * Sets the UI tag for the {@code Person}.
+     * If the tag is None, the tag will not be displayed.
+     */
+    public void setUiTag() {
+        Color tagColor = Color.TRANSPARENT;
+        TagType tagType = person.getTag().value;
+        switch (tagType) {
+        case Professor:
+            tagColor = Color.HOTPINK;
```
Comment (RingoftheKing) [inline src/main/java/seedu/address/ui/PersonCard.java:86]: Very Hot...

File: src/main/resources/view/PersonListCard.fxml [hunk 2]
```diff
@@ -27,10 +30,18 @@
         </Label>
         <Label fx:id="nusId" text="\$first" styleClass="cell_big_label" />
         <Label fx:id="name" text="\$first" styleClass="cell_big_label" />
+
+        <!-- Spacer for aligning the tag at the right -->
+        <Region HBox.hgrow="ALWAYS"/>
```
Comment (RingoftheKing) [inline src/main/resources/view/PersonListCard.fxml:35]: Good comment that details the functionality of the following line

File: src/main/java/seedu/address/ui/PersonCard.java (hunk 1 above)
Comment (Ragnapop) [inline src/main/java/seedu/address/ui/PersonCard.java:86]: Good choice of color!

File: src/main/resources/view/PersonListCard.fxml [hunk 3]
```diff
@@ -27,10 +30,18 @@
         </Label>
         <Label fx:id="nusId" text="\$first" styleClass="cell_big_label" />
         <Label fx:id="name" text="\$first" styleClass="cell_big_label" />
```
Comment (Ragnapop) [inline src/main/resources/view/PersonListCard.fxml:32]: Good edge case handling

**Pull request merged**